lib/index.js

```diff
@@ -276,7 +276,7 @@ Queue.prototype._checkReclaim = function() {
 
   function findOtherQueues(name) {
     var res = [];
-    var storage = self._store.engine;
+    var storage = self._store.getOriginalEngine();
```

Comment:
question: One of the scenarios that triggers a switch to the in memory engine is a failure to `getItem` from localStorage, for example if a user has blocked localStorage access to a particular domain.
In this scenario how do we handle failures to get the reclaim and ack etc?

Reply:
`getItem` just returns `null` if there is an error ([ref](https://github.com/segmentio/localstorage-retry/blob/master/lib/store.js#L43-L53)). The check to see if we can get items from localStorage happens inside of the engine ([ref](https://github.com/segmentio/localstorage-retry/blob/master/lib/engine.js#L36-L61)).

This means that if the user has blocked localStorage, the initial `defaultEngine` will be inMemory. This will mean that this code path will work as expected.